portable/release.sh

```diff
@@ -73,7 +73,7 @@ echo "Creating archive" &&
  echo 'ExtractTitle="Extracting..."' &&
  echo 'GUIFlags="8+32+64+256+4096"' &&
  echo 'GUIMode="1"' &&
- echo 'InstallPath="%'$PROGRAMFILESENV'%\\Git"' &&
+ echo 'InstallPath="%%S\\PortableGit"' &&
```

Comment:
If PROGRAMFILESENV is not used here, its initialization should also get removed from the beginning of the script.